frontend/package.json

```diff
@@ -2,18 +2,11 @@
   "name": "trase-frontend",
   "version": "1.0.0",
   "description": "",
-  "main": "server.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "npm run webpack:server",
-    "dev": "npm run start",
-    "build": "npm run webpack:prod && cp -a public/. dist/",
-    "webpack:prod": "NODE_ENV='production' webpack --config config/webpack.js -p",
-    "webpack:server": "webpack-dev-server --inline --port 8081 --host 0.0.0.0 --content-base public/ --config config/webpack.js",
-    "eslint": "npm run lint-js",
+    "start": "webpack-dev-server --inline --port 8081 --host 0.0.0.0 --content-base public/ --config config/webpack.js",
+    "build": "NODE_ENV='production' webpack --config config/webpack.js -p && cp -a public/. dist/",
```

Comment:
I prefer having separated NPM tasks to have a more readable package.json. Agglomerating tasks into single lines tend to make them long and difficult. I'd keep `"webpack:prod"`